### DISCUSSION TITLE
Relationships between AR and Sequel models

### DISCUSSION THREAD

**Discussion by pjmelling** [janko/sequel-activerecord_connection#28] [Q&A] [answered]
Is it possible to have relationships between ActiveRecord and Sequel models?

**Comment by janko** [comment, marked as answer]
If you mean associations, then no. Active Record explicitly requires that the association model class is an `ActiveRecord::Base` subclass. Sequel doesn't seem to explicitly require it, but it calls methods like `.dataset` that are only available on Sequel models.

**Reply by pjmelling** [reply]
I was trying to get it working and running into those very issues. Just wanted to make sure I wasn't missing something. Thank you for the quick reply. And thank you for the gem!
